src/Applications/RTFRevit/RevitTestClient.cs

```diff
@@ -114,11 +114,23 @@ public static void ShutdownClient()
         /// <param name="fixtureName"></param>
         public static void SendTestInformation(string testName, string fixtureName)
         {
-            if (clientSocket != null)
-            {
-                DataMessage msg = new DataMessage(testName, fixtureName);
-                SendMessage(msg);
-            }
+            DataMessage msg = new DataMessage(testName, fixtureName);
+            SendMessage(msg);
+        }
+
+        internal static void SendTestResultInformation(string testName, string fixtureName, string result, string stackTrace)
+        {
+            TestResultMessage msg = new TestResultMessage(testName, fixtureName, result, stackTrace);
+            SendMessage(msg);
+        }
+
+        /// <summary>
+        /// Sends a message with a line of console text that the text produces
```

Comment:
I have read this 3 times and am still confused 😉

Reply:
haha, sorry, yeah, this is not comprehensible - let me fix that!